src/BaGet.UI/src/config.tsx

```diff
@@ -2,6 +2,10 @@ let config = {
   apiUrl: "__BAGET_PLACEHOLDER_API_URL__"
 };
 
+if (process.env.NODE_ENV === 'test') {
```

Comment:
need this to solve node fetch. It needs a full URL.

Reply:
Could you leave a comment about this?

Also, what do you think of only setting `apiUrl` to `http://localhost` if the `apiUrl` currently is `__BAGET_PLACEHOLDER_`?